docker-compose.yml

```
@@ -3,7 +3,7 @@ services: # 아래에서 설정한 컨테이너들을 하나로 묶어줌
     image: mysql:8.0 # docker mysql 이미지 버전
     container_name: spoton-mysql # 컨테이너 이름
     environment: # mysql 환경변수
-      MYSQL_ROOT_PASSWORD: 1234 # root 비밀번호
+      MYSQL_ROOT_PASSWORD: 97993kim@ # root 비밀번호
```

```suggestion
      MYSQL_ROOT_PASSWORD: 1234 # root 비밀번호
```
따로 설정하지 마시고 기본 설정을 따라주세요.